package.json

```diff
@@ -6,7 +6,6 @@
     "build:lib": "npm run build -w libraries",
     "lint:lib": "npm run lint -w libraries --if-present",
     "test:unit": "npm run test -w libraries/ui-library",
-    "demo:vue": "npm start -w examples/vue",
```

Comment:
@pennal FYI I suggest you do a rebase after this is merged, probably you have some conflicts with your fork